ZenPacks/zenoss/Microsoft/Windows/datasources/EventLogDataSource.py

```diff
@@ -322,9 +322,8 @@ def _makeEvent(evt):
                         'device': config.id,
                         'summary': 'Windows EventLog: Received PowerShell error during event collection',
                         'message': ps_err_msg,
-                        'severity': ZenEventClasses.Warning,
+                        'severity': ds0.severity or ZenEventClasses.Warning,
                         'eventKey': 'EventLogPowerShell: {}'.format(ds0.params.get('eventid', '')),
-                        'eventClass': '/Status/Winrm',
                         'eventClassKey': 'WindowsEventLogWarning',
                     })
                 else:
```

Comment:
can you remove `log.debug(output)` on line 336?  we're already logging the output above